src/rules/selector-descendant-combinator-no-non-space/README.md

```diff
@@ -0,0 +1,32 @@
+# selector-descendant-combinator-no-non-space
+
+Disallow non-space characters for descendant combinators of selectors.
+
+```css
+   .foo .bar .baz { color: pink; }
+/**          ↑          ↑
+ * These are descendant combinators */
+```
+
+This rule ensures that there is only single space for descendant combinators of selectors.
+
```

Comment:
Please somewhere in this description include explicit mention of tabs, newlines, and multiple spaces. I still don't think the purpose of this rule is made immediately obvious by these docs.